src/sequelize/config/config.js

```diff
@@ -5,13 +5,32 @@ module.exports = {
   development: {
     url: process.env.DB_CONNECTION,
     dialect: "postgres",
+    dialectOptions: {
+      ssl: {
+        require: true,
+        rejectUnauthorized: false,
+      },
+    },
```

Comment:
do you think this can allow us to use local database

Reply:
This allow us to connect to the deployed Database so we are no longer use local database. we're using the deployed one.